themes/src/main/resources/theme/keycloak-preview/account/messages/messages_en.properties

```diff
@@ -110,3 +111,6 @@ removeModalTitle=Remove Access
 removeModalMessage=This will remove the currently granted access permission for {0}. You will need to grant access again if you want to use this app.
 confirmButton=Confirm
 infoMessage=By clicking 'Remove Access', you will remove granted permissions of this application. This application will no longer use your information.
+
+#Delete Account page
+deleteAccountSummary=Deleting your account will erase all your data and log you out immediately. 
```

Comment:
I know this might sound far-fetched, but the account console is used by novice users.  We shouldn't tell them it will "erase all your data".  They might think it will wipe the hard drive.

Though I'm a bit hard-pressed to suggest what the text should actually be.  @andymunro?

Reply:
This is actually the message proposed by @andymunro